src/main/webapp/table.js

```diff
@@ -168,4 +197,137 @@ class Table {
     }
     return columnNamesRow;
   }
-}
\ No newline at end of file
+
+  addPaginationButtons(thisTableDiv, pageInfoString) {
+    const buttonDiv = document.createElement("div");
+    buttonDiv.id = "button-div-" + this.name;
+
+    const nextButton = document.createElement("button");
+    nextButton.innerHTML = "Next";
+    nextButton.id = "next-button-" + this.name;
+    const id = this.id;
+    nextButton.onclick = function() { nextPage(id); }
```

Comment:
I believe you can just call this.nextPage() here. Also try using an arrow funciton `nextButton.onclick = () => this.nextPage()`

Reply:
Done!